.github/workflows/create-s3-bucket-w-tofu.yml

```
@@ -0,0 +1,93 @@
+        type: choice
+        options: [AES256, aws:kms, none]
+        default: AES256
+      kms_key_arn:
+        description: "(Optional) KMS Key ARN when encryption=aws:kms"
+        required: false
+        type: string
+      extra_tags_json:
+        description: "(Optional) JSON map of extra tags, e.g. {\"Env\":\"dev\",\"Owner\":\"DevOps\"}"
+        required: false
+        type: string
+    pull_request:
```

actionlint (1.7.7)

expected "inputs" key for "workflow_dispatch" section but got "pull_request"

(syntax-check)